moveit_setup_assistant/src/tools/compute_default_collisions.cpp

```diff
@@ -366,7 +366,7 @@ void computeConnectionGraph(const moveit::core::LinkModel* start_link, LinkGraph
           temp_list.push_back(adj_it);
         }
 
-        // Make all preceeding and succeeding links to the no-shape link fully connected
+        // Make all proceeding and succeeding links to the no-shape link fully connected
```

Comment:
Thank you for cleaning up the spelling in this package.  I wouldn't be surprised if there are many spelling errors all over the repo.  Would you like to make a PR to fix the spelling in the whole repo?